src/lolex-src.js

```diff
@@ -611,32 +616,25 @@ function createClock(now, loopLimit) {
 }
 exports.createClock = createClock;
 
-exports.install = function install(target, now, toFake, loopLimit) {
-    var i, l;
-
-    if (target instanceof Date) {
-        toFake = now;
-        now = target.getTime();
-        target = null;
-    }
-
-    if (typeof target === "number") {
-        toFake = now;
-        now = target;
-        target = null;
-    }
-
-    if (!target) {
-        target = global;
-    }
+/**
+ * @param config {Object} optional config
+ * @param config.target {Object} the target to install timers in (default `window`)
+ * @param config.now {number|Date}  a number (in milliseconds) or a Date object (default epoch)
+ * @param config.toFake {string[]} names of the methods that should be faked.
+ * @param config.loopLimit {number} the maximum number of timers that will be run when calling runAll()
+ */
+exports.install = function install(config) {
+    config = typeof config !== "undefined" ? config : {};
 
-    var clock = createClock(now, loopLimit);
+    var i, l;
+    var target = config.target || global;
+    var clock = createClock(config.now, config.loopLimit);
```

Comment:
The same [as today](https://github.com/fatso83/lolex/blob/10c053fa919141297ef77703468692b7ae6a7fc3/src/lolex-src.js#L88). It is converted to 0 down the line. 

I am not a super-fan of the current digging-down-into-the-code to see if not passing a number is allowed or not, so adding defaults higher up along with strict checking further down, is more my style.